### DISCUSSION TITLE
蓝灯邮箱收不到验证码，反馈了一个月了，也不给回复，也不给解决方案，你们就是这么做产品的吗，只管收钱不管解决问题

### DISCUSSION THREAD

**Discussion by WeiChenUm** [getlantern/lantern-forum-cn#825] [General]
上个月就反馈了收不到验证码的问题，国内邮箱收验证码被限制了，那么解决方案呢？反馈了那么久也不给回复，到底要怎么解决，就这么一直拖着吗？

收钱倒是挺顺畅，解决问题一直拖，用了几年蓝灯了，要么就支持换邮箱

还有我这一个月的时间无法正常使用，请把这一个月的时间给我加上去

![355053925-2900b095-9423-4bf1-9fcf-22c7dc1c61f6](https://github.com/user-attachments/assets/3115d6f0-2093-4759-9507-77b1364a4524)

**Comment by MaxMKMax** [comment]
@WeiChenUm 验证码收不到是因为被墙挡住了. 非常抱歉影响了您的体验感. 我已经帮您提交问题上去了. 尽快在两天内给您解决问题. 非常谢谢您支持蓝灯.
